src/components/GetTasks.js

```diff
@@ -0,0 +1,52 @@
+import { useEffect, useState } from "react";
+import axiosInstance from "../api/axios";
+
+const GetTasks = ({ getTask, todos }) => {
+  const [mount, setMount] = useState(false)
+
+  const handleDelete = async (id) => {
+    try {
+      await axiosInstance.delete(id + "/");
+      getTask();
+    } catch (error) {
+      console.log(error);
+    }
+  };
+  
+  useEffect(() => {
+    if(!mount){
+      setMount(true)
+      getTask();
+    }
+  }, [getTask, mount]);
```

Comment:
Why?

Reply:
To avoid re-rendering issue and only call function when the component is mounted